### DISCUSSION TITLE
Error: Claim not Found

### DISCUSSION THREAD

**Discussion by vladimir9195** [FBoucher/TinyBlazorAdmin#50] [Q&A]
Good day Frank,

I installed URLShortner + TinyBlazorAdmin on our Azure environment. I have no issues using TinyBlazorAdmin portal. Can create/delete URLs. But then I try to make a direct call to UrlShortener from PostMan I have an error 400 Bad Request with a reply body message: Claim not Found.

My postman request is next: 
Post request to https://xxxxxxx.azurewebsites.net/api/UrlShortener
Header: 
x-functions-key:XXXXXXXXXXXXXXXXXXXXXXXXXXXX==(  this is a key from azure function)
Content-Type:application/json
Body:

{
    "url": "http://www.frankysnotes.com/wer?werr",
    "vanity": ""
}

Reply:
{
    "message": "Claim not Found",
    "statusCode": 400
}

I will appreciate your help.

Thank you,
Vladimir

**Comment by fboucher** [comment]
@vladimir9195 you need an Azure Active Directory (AAD) token.
The Azure Function is secured with it to avoid a security breach.

Have a look at this doc it will give you an idea: https://docs.microsoft.com/en-us/graph/use-postman
